shell/platform/android/io/flutter/embedding/android/FlutterView.java

```diff
@@ -538,7 +539,7 @@ protected void onDetachedFromWindow() {
   @TargetApi(API_LEVELS.API_28)
   protected void setWindowInfoListenerDisplayFeatures(WindowLayoutInfo layoutInfo) {
```

Comment:
I dont understand why only the display cutout features need to be moved and not all the features in this method. 

Unrelated to your change it is weird that this had a targetApi of 28 and still did an api check inside the method.

Reply:
My understanding is that this listener is triggered (only) on changes to [`DisplayFeature`](https://developer.android.com/reference/kotlin/androidx/window/layout/DisplayFeature), which include hinges and display folds but (from @yaakovschectman and my manual testing) not camera cutouts (or other top of device sensors).

Those are instead reported as [`WindowInsets`](https://developer.android.com/reference/android/view/WindowInsets#getDisplayCutout()) with a nonzero element reported by `getDisplayCutout()`.

Flutter was conflating both of these two distinct groups into a `List<FlutterRenderer.DisplayFeature>`, but the list would only get updated in the case of changes to the former group, because underlying updates to the latter didn't trigger the listener.